api/src/services/block-cursor.ts

```diff
@@ -1,16 +1,14 @@
 import { BlockCursor } from "../models/block-cursor";
 
 class BlockCursorService {
-  async findOrCreateLatestBlockCursor(
-    eventName: string,
-    latestBlockHeight: number
-  ) {
+  private readonly cursorLabel: string = "BLOCK_CURSOR";
```

Comment:
What's the purpose of the label if there's also a cursor ID? It looks like there was originally supposed to be one cursor per event type, but now there's one global cursor. Is that right?

I would make `cursorLabel` a parameter and allow `BlockCursorService` to maintain multiple cursors. Even if that's the case, I think it's totally okay for the event handler to use the same cursor for all events right now.

Reply:
I thought about this, but didn;t see the sense of maintaining 2 cursors, since we get all the events we're interested in in a single query?

The label is there, because we don't know the id beforehand, so can't use it in the JS to query the table for the cursor.